src/components/common/Modal/index.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
+import { Common } from '@styles/globalStyle';
+import { ReactNode } from 'react';
+
+interface Props {
+  size?: 'small' | 'big';
+  isOpen: boolean;
+  title: string;
+  content: ReactNode;
+}
+
+const Modal = ({ size = 'small', isOpen, title, content }: Props) => {
+  return (
+    <Wrapper>
+      {isOpen && (
+        <Overlay>
+          <ModalBox size={size}>
+            <Header>
+              {size === 'small' ? (
+                <SmallHeaderBar>{title}</SmallHeaderBar>
+              ) : (
+                <BigHeaderBar>{title}</BigHeaderBar>
+              )}
```

Comment:
ModalBox처럼 style component에서 분기를 줘서 처리를 해줘도 좋을거 같네요 

```suggestion
                <HeaderBar size={size}>{title}</BigHeaderBar>
```